dozer/cogs/modmail.py

```diff
@@ -74,16 +75,20 @@ async def send_modmail_embeds(self, source_channel, message_content, received_fi
         if received_files is None:
             received_files = []
         lookup = await ModmailThreads.get_by(user_thread=source_channel)
+        is_user_thread = True
         if len(lookup) == 0:
             lookup = await ModmailThreads.get_by(mod_thread=source_channel)
+            is_user_thread = False
         mod_thread = self.bot.get_channel(lookup[0].mod_thread)
         user_thread = self.bot.get_channel(lookup[0].user_thread)
         guild = user_thread.guild
 
         to_send = message_content
+        red_or_green = discord.Color.green() if is_user_thread else discord.Color.red()
```

Comment:
NIT: This seems like a weird name for a variable. Might just call it `color` or, even better, assign this inline.